src/components/context/ExperienceContext.tsx

```diff
@@ -109,19 +110,19 @@ export default ({
           }
         });
       });
-  }, [firebase, setReactions, experienceName]);
+  }, [firebase, setReactions, venueName]);
 
   const addReaction = useCallback(
     (newReaction: Reaction) => {
       firebase
         .firestore()
-        .collection(`experiences/${experienceName}/reactions`)
+        .collection(`experiences/${venueName}/reactions`)
         .add(newReaction);
     },
-    [firebase, experienceName]
+    [firebase, venueName]
   );
 
-  const store = { reactions, addReaction };
+  const store = { reactions, addReaction, venueName };
```

Comment:
Why not add a name to the Venue, that's the same value as the id? I don't think we should have this information split.

Reply:
Yep and we can use the document ID as the venue name. That will ensure it’s unique

Reply:
Yep and we can use the document ID as the venue name. That will ensure it’s unique